pkgs/by-name/ya/yandex-music/ym_info.json

```diff
@@ -1,6 +1,6 @@
 {
-  "version": "5.57.0",
-  "exe_name": "Yandex_Music_x64_5.57.0.exe",
-  "exe_link": "https://music-desktop-application.s3.yandex.net/stable/Yandex_Music_x64_5.57.0.exe",
-  "exe_hash": "sha256-FIr77eH0ksKSLDJBbyTSd/Qkwd1UFc/VFJ3GEnbOD90="
+  "version": "5.63.1",
+  "exe_name": "Yandex_Music_x64_5.63.1.exe",
+  "exe_link": "https://music-desktop-application.s3.yandex.net/stable/Yandex_Music_x64_5.63.1.exe",
```

Comment:
If the tag is unstable, can we assume this one is stable?

Reply:
This is more stable. This is binary distribution by Yandex, the only problem with which can be - is its deletion on Yandex servers.

While tags are for the [repacker](https://github.com/cucumber-sp/yandex-music-linux) (from exe to linux) project, the release circle of which is:
* automatically update hashes and tag once new binary appeared on servers
* fixes by hand and moving tags on new fixes commits.